syz-manager/mgrconfig/mgrconfig.go

```diff
@@ -34,6 +34,8 @@ type Config struct {
 	Hub_Addr   string
 	Hub_Key    string
 
+	Email_addr string
```

Comment:
Name it as other fields in this struct -- `Email_Addr`, and add a comment that manager will send mail for new crashes to this address and that this is optional. Also update `docs/configuration.md`.